### PR TITLE
404 styling and encoding.

### PR DESCRIPTION
**Pull request by andorp**
Fixes: #146 
Fixe: #208

### CODE REVIEW

File: src/Frontend/Middleware.hs [hunk 1]
```diff
@@ -37,11 +37,14 @@ catch404 devMode app req cont = app req $ \resp -> cont $ f resp
         else responseBuilder status headers builder
       where
         status  = responseStatus resp
-        headers = responseHeaders resp
         builder = Builder.byteString . cs
                 . (`runReader` whereToGetTheLangValue) . renderTextT . toHtml
                 $ PublicFrame Page404 [] devMode
 
+        setContentType ("Content-Type", "text/plain") = ("Content-Type", "text/html;charset=utf-8")
```
Comment (fisx) [inline src/Frontend/Middleware.hs]: misleading name (even if local).  you are setting the content type *encoding*, not the *type*.  i'll change it to something.

Comment (fisx) [inline src/Frontend/Middleware.hs]: ah, type also changes to html, sorry.  alsmost correct then, but `repair` is still better than `set` i think.

**Comment by fisx** [comment]
LGTM!

can be merged if you and travis agree.

**Comment by andorp** [comment]
LGTM

**Pull request merged**